mailbox/store/src/main/java/org/apache/james/mailbox/store/search/ListeningMessageSearchIndex.java

```diff
@@ -132,7 +132,7 @@ protected Mono<Void> handleAdded(MailboxSession session, Mailbox mailbox, Added
 
     private Flux<MailboxMessage> retrieveMailboxMessages(MailboxSession session, Mailbox mailbox, MessageRange range) {
         return factory.getMessageMapper(session)
-            .findInMailboxReactive(mailbox, range, FetchType.FULL, UNLIMITED);
+            .findInMailboxReactive(mailbox, range, FetchType.HEADERS, UNLIMITED);
```

Comment:
Wouldn't that make the search on the mail body unavailable though? I don't see why a client using James would find that acceptable?